### PR TITLE
FE-40: Create NavbarLink and fill value in Booking page

### PR DESCRIPTION
**Pull request by thangduongnfq**

### CODE REVIEW

File: src/redux/slices/bookingFlightsSlice.js [hunk 1]
```diff
@@ -1,42 +1,72 @@
...
+export const createBookingFlight = createAsyncThunk(
+  'flight/createBooking',
+  async (params) => {
+    const respone = await flightAPI.createATicket(params)
+    return respone.data
+  }
+)
+
 const bookingFlightsSlice = createSlice({
   name: 'filterSlice',
```
Comment (DucTranNFQ) [inline src/redux/slices/bookingFlightsSlice.js:46]: I don't know why bookingFlightsSlice is named filterSlice. Is it a mistake ? 😁

**Review by DucTranNFQ — APPROVED** [review]
Good job bro~!

File: src/redux/slices/bookingFlightsSlice.js (hunk 1 above)
Comment (zevik7) [reply src/redux/slices/bookingFlightsSlice.js:46]: He copy old code but forgot change their name :))

**Review by zevik7 — COMMENTED** [review]
fix conflict first bro :))

**Comment by thangduongnfq** [comment]
fixed !! 🎉🎉

**Review by gioiemhuynh — APPROVED** [review]
That't OK!

**Pull request merged**